src/components/ViewContainer/getCRUDActions.js

```diff
@@ -0,0 +1,62 @@
+export const getCRUDActions = ({
+  entityKey,
+  mutator: {
+    [entityKey]: resourceMutator,
+    restoreDefaults,
+  },
+  onSuccess,
+  onError,
+}) => {
+  const createRecord = record => resourceMutator.POST(record)
+    .then(response => {
```

Comment:
Since you refactoring stuff from S&S I think it makes sense to rewrite to use async/await syntax for consistency.